packages/ibm-products/src/components/Tearsheet/_tearsheet.scss

```
@@ -270,11 +272,15 @@ $motion-duration: $duration-moderate-02;
   .#{$block-class}__main {
     display: flex;
     flex-direction: row;
-    background-color: $layer;
+    background-color: $background;
     grid-column: 1 / -1;
     grid-row: 1 / -1;
   }
 
+  .#{$block-class}--narrow .#{$block-class}__main {
```

Since this whole style block is nested in `--tearsheet` and the `--tearsheet--narrow` selector is used on the same element, you'll need to use the sass parent selector (`&`) here.

```suggestion
  &.#{$block-class}--narrow .#{$block-class}__main {
```

---

Thanks Jan. Updated as suggested.